src/fastoad/gui/mission_viewer.py

```diff
@@ -73,11 +75,22 @@ def display(self):
 
         self._output_widget = widgets.Output()
 
+        flight_point_units = FlightPoint.get_units()
+
         # By default ground distance
-        self._x_widget = widgets.Dropdown(value=keys[2], options=keys)
+        unit_ground_distance = flight_point_units["ground_distance"]
+        column_ground_distance = f"ground_distance [{unit_ground_distance}]"
+        idx_ground_distance = (
+            keys.to_list().index(column_ground_distance) if column_ground_distance in keys else 3
+        )
+        self._x_widget = widgets.Dropdown(value=keys[idx_ground_distance], options=keys)
```

Comment:
I would suggest this more pythonic way:
```suggestion
        x_label = column_ground_distance if column_ground_distance in keys else keys[3]
        self._x_widget = widgets.Dropdown(value=x_label, options=keys)
```
Besides, I would be tempted to add the method `_get_label(self, keys, quantity_name, default_idx)` to replace lines 81-85 by `x_label = self._get_label(keys, "ground_distance", 3)` and lines 90-92 by `y_label = self._get_label(keys, "altitude", 1)`.
That would require to include line 78 in the method, and then do it twice, but I think it's acceptable.

Reply:
Done in 7e1f5284